src/utils/download.ts

```diff
@@ -52,3 +54,16 @@ export async function downloadProject(project: Project, uri?: string) {
     document.body.removeChild(link);
   });
 }
+
+async function addDevKey(queue: Array<{filePrefix: string; file: File}>, uri: string) {
+  const app = (window as any).app;
+  const contractName = `studio-${uri}`;
+  const keyPair = await app.state.keyStore.getKey(contractName);
+  if (keyPair && keyPair.getPublicKey() && keyPair.getSecretKey()) {
+    const networkId = "unknown"; // TODO: query network id from node
+    const accountInfo = new AccountInfo(contractName, new KeyPair("a", "b"), networkId);
```

Comment:
`new KeyPair("a", "b")` this looks like accidentally hardcoded

Reply:
that was an old version